Source/LibationWinForms/ProcessQueue/TrackedQueue[T].cs

```diff
@@ -226,5 +226,14 @@ public void Enqueue(T item)
 				QueuededCountChanged?.Invoke(this, _queued.Count);
 			}
 		}
+
+		public void Enqueue(IEnumerable<T> item)
+		{
+			lock (lockObject)
+			{
+				_queued.AddRange(item);
+				QueuededCountChanged?.Invoke(this, _queued.Count);
```

Comment:
Firing an event inside a lock can deadlock -- please move event outside of lock

Reply:
Done.

Reply:
Thanks for finding those others also